tests/dragonfly/pymemcached_test.py

```diff
@@ -122,3 +123,14 @@ def test_version(memcached_client: MCClient):
     stats = memcached_client.stats()
     version = stats[b"version"].decode("utf-8")
     assert version.startswith("v") or version == "dev"
+
+
+@dfly_args(DEFAULT_ARGS)
+def test_flags(memcached_client: MCClient):
+    for i in range(1, 20):
+        flags = random.randrange(50, 1000)
+        memcached_client.set("a", "real-value", flags=flags, noreply=True)
+
+        res = memcached_client.raw_command("get a", "END\r\n").split()
+        if len(res) > 0:
+            assert res[2].decode() == str(flags)
```

Comment:
1. The python client has no way to get the flags, yes, I grepped flags in the source
2. I have no clue why replies dissapear when I use raw connections, I've spent already ten times more time on it than on the fix

Reply:
@dranikpg  I did not understand (2) - what do you mean by that?

Reply:
> I did not understand (2) - what do you mean by that?

sometimes I get an empty reply, that's why I check for it